### DISCUSSION TITLE
Starting Chromium with CORS extension enabled

### DISCUSSION THREAD

**Discussion by db-dhananjayb** [seleniumbase/SeleniumBase#1959] [Q&A] [answered]
I want to launch chromium browser with the CORS extension "Moesif Origin & CORS Changer" enabled and further test local website with various CORS options enabled. Can someone help me how to achieve this with SeleniumBase. Alternatively I will also appreciate if there is a better proven way to play with CORS options by playing with profiles/capabilities. I can run the test using pytest --disable-ws option however I am looking for a more fine grained control with code.
I quickly searched for discusions related with 'CORS' and could not find anything.
Thanks a lot in advance.

**Comment by mdmintz** [comment, marked as answer]
Use these command-line options to load a chromium extension at the start of tests:

```
--extension-zip=ZIP  (Load a Chrome Extension .zip|.crx, comma-separated.)
--extension-dir=DIR  (Load a Chrome Extension directory, comma-separated.)
```

For CORS-specific questions, see StackOverflow.
